composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "symfony/webpack-encore-bundle",
+    "description": "Integration with your Symfony app & Webpack Encore!",
+    "type": "symfony-bundle",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\WebpackEncoreBundle\\": "src"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Symfony\\WebpackEncoreBundle\\Tests\\": "tests/"
+        }
+    },
+    "require": {
+        "php": ">=7.1.3",
```

Comment:
We do not support PHP8, lets do `^7.1.3` (even tough PHP is not following Semver)